src/schemas/json/web-manifest.json

```diff
@@ -1,9 +1,9 @@
 {
   "title": "JSON schema for Web Application manifest files",
-  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
SchemaStore.org and the IDEs and editors supporting it, only work with JSON Schema Draft v4, so this won't work. It appears that only the `$comment` property is the only v7 concept in the PR

Reply:
Revert schema to draft-4